apps/oc_chef_wm/Makefile

```diff
@@ -21,7 +21,7 @@ DIALYZER_DEPS = deps/chef_authn/ebin \
 # :'(
 DEPS_PLT = oc_chef_wm.plt
 # TODO automate managing this revision
-EC_SCHEMA_VERSION = 2.6.0
+EC_SCHEMA_VERSION = 31048bf983361c7b4a3f19e8aeb836a31c95cee6 # temporary to make CT tests pass in travis
```

Comment:
This also should work if you set to a branch name, if that makes it easier to manage.